opentech/apply/funds/templates/funds/applicationsubmission_simplified_detail.html

```diff
@@ -0,0 +1,58 @@
+{% extends "base-apply.html" %}
+
+{% load wagtailcore_tags statusbar_tags %}
+
+{% block title %}{{ object.title }}{% endblock %}
+
+{% block body_class %}{% endblock %}
+
+{% block content %}
+{% if object.round.specific.is_sealed %}
```

Comment:
We probably want to avoid this check entirely - lets restrict this view to when there is a project in existence only so as not to trip up on issues like this.

Reply:
Fixed and rebased.